js/js.dce/src/org/jetbrains/kotlin/js/dce/ReachabilityTracker.kt

```diff
@@ -19,11 +19,13 @@ package org.jetbrains.kotlin.js.dce
 import org.jetbrains.kotlin.js.backend.ast.*
 import org.jetbrains.kotlin.js.dce.Context.Node
 import org.jetbrains.kotlin.js.inline.util.collectLocalVariables
+import java.util.concurrent.atomic.AtomicInteger
```

Comment:
[minor] unused import

Reply:
@anton-bannykh Vlad is on vacation, have you ran the tests to determine this fix is good?

Reply:
@morj Sorry for the delay, forgot to continue this task after the tests were finished. About 8 tests failed, but that could be just flakyness, re-running the test & investigating.

Reply:
@anton-bannykh can you also please fix minor issues like unused imports yourself since Vlad is still on vacation?

Reply:
Removed unused import